SpywareFilter/sections/mobile.txt

```diff
@@ -176,6 +176,7 @@ inapps.appsflyersdk.com^
 !
 ! NOTE: Regular rules
 !
+||newsstat.dallasnews.com^
```

Comment:
Request happens on desktop too, so please move to tracking_servers_firstparty

Reply:
Done